stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1363,6 +1370,9 @@ public enum TestRunPredicate : CustomStringConvertible {
     case linuxAny(reason: let reason):
       return "linuxAny(*, reason: \(reason))"
 
+    case androidAny(reason: let reason):
+      return "AndroidAny(*, reason: \(reason))"
```

Comment:
Good catch! These were originally `LinuxAny`, etc., before they were converted to lowercase `linuxAny`. I missed this spot when rebasing these changes. Thanks for pointing this out!